source/StellarObject.cpp

```diff
@@ -155,6 +156,42 @@ double StellarObject::Distance() const
 
 
 
+void StellarObject::UpdateDistanceVisibility(const Ship *flagship)
```

Comment:
```suggestion
// Change the distances an object is visible from, based on ship attributes.
void StellarObject::UpdateDistanceVisibility(const Ship *flagship)
```

Reply:
gotta say I am not a fan of these duplicate comments